Surf MVP Application.xctemplate/Makefile

```
@@ -1,8 +1,7 @@
-## Installs all dependencies
 init:
 	# Install bundler if not installed
 	if ! gem spec bundler > /dev/null 2>&1; then\
-  		echo "bundler gem is not installed!";\
+  		echo "bundler gem is not installed! Run gem install bundler to install it";\
   		-sudo gem install bundler;\
```

Здесь нужно явно указывать версию, которую нужно установить
```suggestion
  		-sudo gem install bundler -v "1.17.3";\
```